### PR TITLE
Fixed font size and widget

### PR DESCRIPTION
**Pull request by PartishtaP**
The changes which were required are made.
For example adjusting the font size so that it doesn't overlap.
Text size has been reduced which is suitable for mobile application.

### CODE REVIEW

**Review by muzammildafedar — CHANGES_REQUESTED** [review]
Good fix, Could you please keep the previous heading 1,2,3,4,5,6 for desktop also ?

File: lib/styles/fonts.dart [hunk 1]
```diff
@@ -51,6 +51,49 @@ class AppTextStyles {
...
+    fontSize: 16,
+  );
+
+  static const TextStyle heading6Mobile = TextStyle(
+    fontFamily: 'Poppins',
+    fontWeight: FontWeight.bold,
+    color: Colors.black,
+    fontSize: 14,
+  );
+
```
Comment (muzammildafedar) [inline lib/styles/fonts.dart:96]: Restore previous headings for desktop here

Comment (PartishtaP) [reply lib/styles/fonts.dart]: **I HAVE MADE THE STARED CHANGES.
AS WANTED, THE DESKTOP HEADINGS HAVE BEEN RESTORED**

Comment (PartishtaP) [reply lib/styles/fonts.dart]: **THE PREVIOUS DESKTOP HEADINGS HAVE BEEN RESTORED**

**Comment by muzammildafedar** [comment]
The changes do not match the issue accordingly. Closing this PR.

**Pull request closed without merging**